operators/test/e2e/apm/checks_k8s.go

```diff
@@ -35,7 +35,7 @@ func CheckApmServerDeployment(stack Builder, k *helpers.K8sHelper) helpers.TestS
 			var dep appsv1.Deployment
 			err := k.Client.Get(types.NamespacedName{
 				Namespace: params.Namespace,
-				Name:      stack.ApmServer.Name + "-apm-server",
+				Name:      stack.ApmServer.Name + "-apm",
```

Comment:
Any particular reason for moving away from `apm-server`?

Reply:
At first it was not intentional, I did it when I was working on the naming. I thought it was not so bad since "-server" does not bring much value. But since the name of the container is `apm-server` I'm ok to keep it 😃